go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/apache/thrift v0.13.0
 	github.com/aws/aws-sdk-go v1.34.0
 	github.com/benbjohnson/clock v1.0.3
+	github.com/cenkalti/backoff v2.2.1+incompatible
```

Comment:
This is a scary dependency (because it says "`+incompatible`", and because we have `v3.2.2` below).  Can we use just a single version?

Reply:
Removed.